src/newrelic.coffee

```diff
@@ -167,6 +177,44 @@ plugin = (robot) ->
 
   start_violations_polling(robot)
 
+  getCfgovDeploys = (deployEnv, parentResolve, page = 1, deploys) ->
+    deploys ?= _.fromPairs(
+      _.zip deployEnv.targets, _.map(deployEnv.targets, (e) -> null)
+    )
+
+    maxPagesToRequest = 6
```

Comment:
Is this pagination number based on anything having to do with the number of possible deployments?